### DISCUSSION TITLE
How to add new libraries permanently mounting them?

### DISCUSSION THREAD

**Discussion by ericzolf** [jgraph/docker-drawio#102] [General]
In order to be able to add my own libraries to draw.io, I tried to map them within the container with:

```console
podman run --rm --name="drawio" --detach \
		--pull=newer -v /home/myuser/drawio-libs/libs:/usr/local/tomcat/webapps/draw/stencils/zdrawio:Z,U \
		-p 8080:8080 -p 8443:8443 jgraph/drawio
```

drawio-libs being a git clone from https://github.com/jgraph/drawio-libs.git

It works in the sense that I can see the directory and its content from within the container, with the correct tomcat access rights, but I don't get any additional library in the UI of draw.io. Anything I might be missing here?

**Comment by ravensorb** [comment]
@ericzolf any luck with this -- i am looking to do the same thing.

**Reply by ericzolf** [reply]
I didn't follow up on this and, as you can see, nothing came back.
